pydatalab/pydatalab/routes/v0_1/items.py

```diff
@@ -315,6 +319,12 @@ def _create_sample(sample_dict: dict, copy_from_item_id: Optional[str] = None) -
             401,
         )
 
+    if generate_id_automatically and sample_dict["item_id"]:
+        LOGGER.warning(
+            f"""generate_id_automatically = true was provided to _create_sample(), but an item_id of "{sample_dict['item_id']}" is included in the sample_dict.
+            The provided item_id will not be used."""
+        )
```

Comment:
I've changed this to return a 400 response, since I think that if these incompatible settings are provided it probably indicates a bug in the request and its better to "fail fast" than quietly swallow it.